target/linux/ath79/image/generic.mk

```diff
@@ -1046,3 +1046,13 @@ define Device/zbtlink_zbt-wd323
 		     kmod-usb-serial kmod-usb-serial-cp210x uqmi
 endef
 TARGET_DEVICES += zbtlink_zbt-wd323
+
+define Device/zsun_sd100
+  ATH_SOC := ar9331
+  DEVICE_VENDOR := Zsun
+  DEVICE_MODEL := SD100
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-storage
+  IMAGE_SIZE := 12032k
+  SUPPORTED_DEVICES += zsun-sd100
```

Comment:
This can be dropped as there is no support in ar71xx.

Reply:
Sad to see ar71xx patch was not accepted - dropped line.